actors/verifreg/src/lib.rs

```diff
@@ -436,6 +444,8 @@ impl Actor {
             )
```

Comment:
change the name of the above resolve_to_id_addr to resolve_to_id or resolve_addr_to_id or resolve_to_actor_id

Reply:
change comment "failed to resolve addr {} to ID"